src/components/BottomNavbar/BottomNavbar.tsx

```diff
@@ -120,8 +121,8 @@ const BottomNavbar = ({ title }: { title: string }) => {
             title={themeSelection ? "ArrowIcon" : "JournalIcon"}
           />
           {themeSelection ? <p>Next</p> : <p>{t("Journal")}</p>}
-          {isAddBtnVisible && <GlobalAddBtn add={title} />}
         </button>
+        {isAddBtnVisible && <GlobalAddBtn add={title} />}
```

Comment:
hi @vinaybadgujar102  please put this back inside the journal button. It was done intentionally. as we have to keep the Add button right above to the journal icon. if you move this button out we will loose the alignment on mobile screens. for example this device
![image](https://github.com/tijlleenders/ZinZen/assets/39093015/47ed6520-9e8f-4949-a7a3-912f14d33c40)